cli-plugins/plugin/plugin.go

```diff
@@ -46,6 +46,22 @@ func RunPlugin(dockerCli *command.DockerCli, plugin *cobra.Command, meta manager
 				opts = append(opts, withPluginClientConn(plugin.Name()))
 			}
 			err = tcmd.Initialize(opts...)
+			ogRunE := cmd.RunE
+			if ogRunE == nil {
+				ogRun := cmd.Run
+				//nolint:unparam // necessary because error will always be nil here
+				ogRunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
Ugh, it's weird that we need that nolint. I did a little digging and it's a bug in combination of golangci-lint and unparam: https://github.com/golangci/golangci-lint/issues/1379 which is tricky to fix 😞

Unfortunately, it seems unlikely to be fixed.. But probably wouldn't hurt to link to that issue in the comment.

Reply:
Will do!